vo_models/uws/types.py

```diff
@@ -7,41 +7,63 @@ class ErrorType(str, Enum):
     """Enum for error types."""
 
     TRANSIENT = "transient"
+    """The error is transient and the job may be rerun."""
     FATAL = "fatal"
+    """The error is fatal and the job may not be rerun."""
```

Comment:
FYI, this format was apparently proposed and rejected in `PEP 224`. I saw this briefly discussed again in PEP 727 here. https://peps.python.org/pep-0727/#string-under-definition